asset/teraslice_kafka_reader/index.js

```diff
@@ -488,9 +496,18 @@ function schema() {
     };
 }
 
+function crossValidation(job) {
```

Comment:
This should be noted as something to be removed once we complete the transition.

Reply:
Sure